solr/solrj/src/java/org/apache/solr/common/params/ShardParams.java

```diff
@@ -28,6 +28,9 @@
 public interface ShardParams {
   /** the shards to use (distributed configuration) */
   String SHARDS = "shards";
+
+  /** UUID of the query */
+  String QUERY_ID = "queryID";
```

Comment:
Why not `queryUUID` (and reference the same constant as in other places)?

Reply:
Will that not cause confusion, since this is the generic queryID (generated for every cancellable query) and the other parameter references the queryID the incoming cancellation/list request wants to deal with?